src/js/utils/BetaOptInUtil.js

```diff
@@ -0,0 +1,85 @@
+import Util from "./Util";
+
+// SDK sets/reads this property within package config.
+const BETA_OPTIN_PROPERTY_NAME = "beta-optin";
+const DOT_NOTATION_SCHEMA_PATH = "properties.service.properties.beta-optin";
+
+const BetaOptInUtil = {
+  /**
+   * Finds beta-optin property within cosmos package config schema.
+   *
+   * @param  {Object} configSchema - cosmos package config schema
+   * @return {Boolean} Returns true if configSchema contains beta-optin property
+   */
+  isBeta(configSchema) {
```

Comment:
Let's rename this method to emphasize that this util checks wether a package is beta. Please adjust the comment as well.  #style #naming